### ISSUE TITLE
Style differently urgent/low priority notifications

### ISSUE THREAD

**Issue by emersion** [emersion/mako#3]
Maybe implement some sort of filtering system, e..g to be able to style differently notifications coming from a specific app?

Depends on #7, I don't want to add CLI flags for this.

**Comment by vilhalmer** [comment]
I would propose implementing this as a combination of mpv's concept of profiles and sway's criteria syntax. So, for example, in this configuration Firefox notifications inherit the global margin and padding, but are orange. Anything with high urgency that also has at least one action defined has red text:
```
margin=30
padding=10
background-color=#eeeeee

[app-name="Firefox"]
background-color=#dd9b59

[urgency=high actionable]
text-color=#ff00000
```

This will require pcre support if we want to have matching on the summary and body, but that could wait until a future PR.

**Comment by emersion** [comment]
+1 to this design, looks great
